invenio_override/assets/semantic-ui/js/invenio_override/unlock.js

```diff
@@ -0,0 +1,37 @@
+import $ from "jquery";
+
+async function generateForm() {
+  // get email from `/api/me`
+  let email = "???";
+  const response = await fetch("/api/me");
+  if (response.ok) {
+    const json = await response.json();
+    email = json?.email || "???";
+  }
+
+  // show form
+  $.getScript("https://ub-support.tugraz.at/assets/form/form.js", () => {
```

Comment:
Why do you need this js file? as i remember we have created this to have the ZAMMAD form integrated to the repository.